.travis.yml

```diff
@@ -20,5 +20,9 @@ matrix:
 before_script:
   - export HARMONY_OPTS=$(node --v8-options | grep --only-matching -- '--harmony_generators')
 
+script:
+  - npm run lint
```

Comment:
FYI, I noticed ESLint seems to be throwing errors on older Node versions. (Presumably the latest ESLint is not compatible with them.)

Since ESLint only does static analysis of code and doesn't actually execute it, it technically only needs to run on one version of Node; the result should be the same on all other versions. Not sure what the best way to do that would be in the Travis config, but maybe you could split the lint out into its own job somehow? That'd also have the nice benefit of clearly displaying lint failures separately from test failures in Travis's UI.

Reply:
You might find [build stages](https://docs.travis-ci.com/user/build-stages/) to be well suited for this.

Reply:
Thanks!  I’ll check it out now…

Reply:
Actually I'm not sure build stages are the best idea since they seem to only run later stages if past stages succeed, and I'm not sure we want to do that with lints/tests (the more feedback you get at once the better).

A better option might be to just add the lint as a separate entry in the build matrix: https://docs.travis-ci.com/user/customizing-the-build#Explicitly-Including-Jobs Not sure though; I'll let you decide.